src/docs/admin/references/CalendarServiceReference.adoc

```diff
@@ -0,0 +1,87 @@
+Calendar service is another ProActive micro-service allowing to visualize and manage (edit and delete) all jobs and a filter planned jobs in a calendar client such as Outlook, Thunderbird Lightning, Mac Calendar, etc. Be aware that calendar service could be running only when the ProActive scheduler is already running.
```

Comment:
all jobs and a filter planned jobs -> I don't understand the sentence
calendar service could be running -> should